hutch_python/calc_defaults.py

```diff
@@ -31,23 +31,27 @@ def collect_functions(modules):
                 pass
     return HelpfulNamespace(**functions)
 
+
 # import specific function to have at the top level of the namespace
+funcs = {}
 try:
-    from pcdscalc.diffraction import (bragg_angle, darwin_width)
+    from pcdscalc.diffraction import bragg_angle, darwin_width
+    funcs['bragg_angle'] = bragg_angle
+    funcs['darwin_width'] = darwin_width
 except ImportError:
     print("Failed to import functions from pcdscalc.diffraction")
 
 try:
     from pcdscalc.xray import transmission
+    funcs['transmission'] = transmission
 except ImportError:
     print("Failed to import functions from pcdscalc.xray")
 
 calc_namespace = HelpfulNamespace(
-    darwin_width=darwin_width,
-    bragg_angle=bragg_angle,
-    transmission=transmission,
     be_lens=collect_functions(['pcdscalc.be_lens_calcs']),
     common=collect_functions(['pcdscalc.common']),
     diffraction=collect_functions(['pcdscalc.diffraction']),
-    xray=collect_functions(['pcdscalc.xray'])
+    xray=collect_functions(['pcdscalc.xray']),
```

Comment:
I agree, I think we need to be consistent here: either everything in this module retains backwards compatibility with old versions as implemented above, OR we jettison all of this complexity and simply update the dependency pin on pcdscalc.

Reply:
I don't really have a strong preference here. This module is widely underutilized across the hutches at the moment, so I dont think it will create much chaos to just move forward with a dependency upgrade

Reply:
I personally think either way is totally reasonable